pkg/yurtctl/constants/constants.go

```diff
@@ -186,7 +186,7 @@ spec:
         - /bin/sh
         - -c
         args:
-        - "sed -i 's|__kubernetes_service_host__|$(KUBERNETES_SERVICE_HOST)|g;s|__kubernetes_service_port_https__|$(KUBERNETES_SERVICE_PORT_HTTPS)|g;s|__node_name__|$(NODE_NAME)|g;s|__yurthub_image__|{{.yurthub_image}}|g' /var/lib/openyurt/setup_edgenode && cp /var/lib/openyurt/setup_edgenode /tmp && nsenter -t 1 -m -u -n -i /var/tmp/setup_edgenode {{.action}} {{.provider}}"
+        - "sed -i 's|__node_name__|$(NODE_NAME)|g;s|__yurthub_image__|{{.yurthub_image}}|g;s|__join_token__|{{.joinToken}}|g' /var/lib/openyurt/setup_edgenode && cp /var/lib/openyurt/setup_edgenode /tmp && nsenter -t 1 -m -u -n -i /var/tmp/setup_edgenode {{.action}} {{.provider}}"
```

Comment:
var  `__ndoe_name__` is not used by setup_edgenode, how about to delete it?

Reply:
so does var `provider`

Reply:
Adjusted